bin/papertrail

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-require 'papertrail/cli'
+require './lib/papertrail/cli'
```

Comment:
Why was this added? `./lib` will be added to `$PATH` by rubygems or bundler. `bundle exec papertrail` should work here. At least it does for me but I'm not sure if I have bundler configured special to handle that.